src/pages/inventory/inventory.html

```diff
@@ -26,23 +26,23 @@
   </ion-segment>
 
   <ion-item>
-    <ion-label floating>Brand</ion-label>
+    <ion-label color="primary">Brand</ion-label>
```

Comment:
Here's a screenshot if that did not make sense. The brand is inline, the model is fixed

<img width="312" alt="screen shot 2017-03-18 at 12 26 42 am" src="https://cloud.githubusercontent.com/assets/12487270/24068942/9bd2c1b0-0b71-11e7-9b82-58847d5afb86.png">